resources_pages/install_ds_stack_ubuntu.md

```diff
@@ -164,68 +155,72 @@ Python 3.7.7
 
 ## Essential Python packages
 
-We also prefer to use packages on the conda-forge channel, so we set that to the default by typing the following in the terminal:
+`conda` installs Python packages from different online repositories which are called "channels".
+A package needs to go through thorough testing before it is included in the default channel,
+which is good for stability,
+but also means that new versions will be delayed and fewer packages are available overall.
+There is a community-driven effort called the [conda-forge](https://conda-forge.org/),
+which provides more up to date packages
+To enable us to access the most up to date version of the Python packages we are going to use,
+we will add the more up to date  channel,
+To add the conda-forge channel by typing the following in the terminal:
 
 ```
 conda config --add channels conda-forge
 
 ```
 
-Thus, to install other pacakges individually, we can now use the following: conda install <package-name>. We will install the key packages needed for the start of our program now:
+To install packages individually, we can now use the following command: `conda install <package-name>`. Let's install the key packages needed for the start of our program:
 
 ```
-conda install --yes \
+conda install \
  jupyterlab=2.* \
  numpy=1.* \
  pandas=1.* \
  flake8=3.* \
  black=19.*
 ```
 
+`conda` will show you the packages that will be downloaded,
+and you can press enter to proceed with the installation.
+
 > Note: we will use many more packages than those listed above across the MDS program, however we will manage these using virtual environments (which you will learn about in DSCI 521: Platforms for Data Science).
 
-## Jupyter extensions
-We will be using several Jupyter extensions that help us use Juypter notebooks more smoothly with Git & GitHub. To install them, paste the following in the terminal below:
+## JupyterLab extensions
+
+We will be using a couple of JupyterLab git extensions to facilitate using Jupyter notebooks with Git & GitHub. Install them via the following commands:
```

Comment:
These are not all git extensions technically. Jupytext is kind of like nbconvert on steriods. So I would prefer to not refer to all of these as git extensions.

Reply:
Great, thanks!